MyAddin/Form1.cs

```diff
@@ -0,0 +1,317 @@
+﻿using System;
+using System.IO;
+using System.Windows.Forms;
+using System.Xml;
+
+namespace MyAddin
+{
+    public partial class Form1 : Form
+    {
+        private XmlDocument docXML = new XmlDocument();
+
+        //this is the path where xml files are saved and loaded
```

Comment:
I'd suggest to use multi-line ` /* */ `comment format if its not an inline comment. Looks more professional and better.